recipe/build.sh

```diff
@@ -3,11 +3,25 @@
 # https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
 export CXXFLAGS="$(echo "${CXXFLAGS}" | sed -E 's@-std=c\+\+[^ ]+@@g') -D_LIBCPP_DISABLE_AVAILABILITY"
 
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* .
+
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
+  OPENFST_CROSS_COMPILATION_CONFIGURE_OPTS="--build=${BUILD} --host=${HOST}"
+  echo "Adding cross compile options: ${OPENFST_CROSS_COMPILATION_CONFIGURE_OPTS}"
```

Comment:
where does this echo go?

Reply:
Oh, just a debug message, removed it in the latest commit.